pkg/controller/migplan/migplan_controller.go

```diff
@@ -445,3 +467,62 @@ func (r ReconcileMigPlan) deleteImageRegistryDeploymentForClient(client k8sclien
 	}
 	return nil
 }
+
+func (r ReconcileMigPlan) validateMigAnalytics(plan *migapi.MigPlan) error {
+	log.Info(fmt.Sprintf("Checking if migpanalytics exists or not"))
+	migAnalytics := &migapi.MigAnalyticList{}
+	err := r.List(context.TODO(), k8sclient.MatchingLabels(map[string]string{"migplan": plan.Name}), migAnalytics)
+	if err != nil {
+		if !errors.IsNotFound(err){
+			return err
+		}
+	}
+	for _, migAnalytic := range migAnalytics.Items {
+		if migAnalytic.Spec.AnalyzeExntendedPVCapacity && plan.Spec.Refresh{
+			log.Info("refreshing miganalytics")
```

Comment:
please remove logs unless there is strong reason to keep them

Reply:
+1, it was in my TODO